### ISSUE TITLE
No configuration file retrieved for the name : _config.yaml

### ISSUE THREAD

**Issue by sobkowiak** [cmoulliard/hyla#128]
I have cloned the repo and installed the gem using rake like described in the documentation. Trying to create a new project I'm getting following output and no generated artefacts

```
kso@lenovo:/tmp/test$ hyla new -b MyRevealSlideShow 
No configuration file retrieved for the name : _config.yaml
No configuration file retrieved for the name : _config.yaml
No configuration file retrieved for the name : _config.yaml
No configuration file retrieved for the name : _config.yaml
```

**Comment by cmoulliard** [comment]
I will have a look next week as this error can come from the new change introduced in 1.0.7

**Comment by cmoulliard** [comment]
I know the reason of this issue. I have implemented a logger within hyla where I check if there is a _config.yaml file. With the new option, such a file does not exist. I will investigate what I could do to by pass the logger in this case.

**Comment by georgegoh** [comment]
Any update on this issue? Would be nice to have a work around.

**Comment by cmoulliard** [comment]
The problem has been fixed in hyla 1.0.9.pre.1. You will still see the warning messages but the directory will be created. such messages are displayed is not able to find fo the logger the config file containing the parameters (lie we have for log3j) to log the messages.

```
hyla new -b MyRevealSlideShow
WARN: Unresolved specs during Gem::Specification.reset:
      eventmachine (>= 0.12.9, ~> 1.0)
      ffi (>= 0.5.0, ~> 1.9)
WARN: Clearing out unresolved specs.
Please report a bug if this causes problems.
No configuration file retrieved for the name : _config.yaml
No configuration file retrieved for the name : _config.yaml
No configuration file retrieved for the name : _config.yaml
No configuration file retrieved for the name : _config.yaml
No configuration file retrieved for the name : _config.yaml
No configuration file retrieved for the name : _config.yaml
No configuration file retrieved for the name : _config.yaml
dabou:~/Temp$ cd MyRevealSlideShow/
dabou:~/Temp/MyRevealSlideShow$ ll
total 24
drwxr-xr-x    5 chmoulli  staff   170 Oct 20 18:40 .
drwxr-xr-x  121 chmoulli  staff  4114 Oct 20 18:40 ..
-rw-r--r--    1 chmoulli  staff  6053 Oct 20 18:40 _config.yaml
-rw-r--r--    1 chmoulli  staff   402 Oct 20 18:40 readme.adoc
drwxr-xr-x   16 chmoulli  staff   544 Oct 20 18:40 styles
```
